models/pipeline.js

```diff
@@ -27,6 +27,11 @@ const MODEL = {
         .description('Unique identifier for the application')
         .example('github.com:123456:master'),
```

Comment:
Just to remind, currently we cannot add a new column to existing tables automatically. Probably need to add that column manually if we want to plug it in now.

Related to this issue:
https://github.com/screwdriver-cd/screwdriver/issues/633